cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -27,7 +27,15 @@
       <child link="${name}_link" />
     </joint>
 
-    <link name="${name}_link">
+    <link name="${name}_link"/>
+
+    <joint name="${name}_rotated_joint" type="fixed">
```

Comment:
can we call this `${base}_drive_dir_link`?
I find it better suited as it is named after the respective parameter...